cola/widgets/prefs.py

```diff
@@ -140,6 +142,12 @@ def __init__(self, context, model, parent, source):
         self.show_path = qtutils.checkbox(checked=True)
         self.tabwidth = standard.SpinBox(value=8, maxi=42)
         self.textwidth = standard.SpinBox(value=72, maxi=150)
+        self.logdate = QtWidgets.QLineEdit()
+        tooltip = N_('Similar to git log --date option')
+        self.logdate.setToolTip(tooltip)
+        regex = QtCore.QRegExp(logdate.create_regex())
+        validator = QtGui.QRegExpValidator(regex, self.logdate)
+        self.logdate.setValidator(validator)
```

Comment:
If the setup for the combobox gets too long (it's already longer than the rest) then that should probably be put into a separate `def build_date_formats_combo(parent)` helper function.